queries.sql

```diff
@@ -37,3 +37,5 @@ SELECT owners.id, full_name FROM animals JOIN owners ON animals.owner_id = owner
 EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
 
 EXPLAIN ANALYZE SELECT  vet_id FROM visits WHERE vet_id = 2;
+
+EXPLAIN ANALYZE SELECT full_name, email FROM owners WHERE email = 'owner_18327@mail.com';
```

Comment:
- [ ] Good job so far 👍 but the goal of this project is to improve the performance of your queries. As a result, the queries should have a lower execution time after the indexes have been applied but I can see that all your queries are run using `Seq scan`. If the indexes are applied correctly, you should see an `index scan` on the screenshot. Kindly work on this issue 💯

Reply:
## Project Requirement

- Ran this queries SELECT COUNT(*) FROM visits where animal_id = 4;
   - Before using explain analyse 
   ![analyze-1](https://user-images.githubusercontent.com/70589265/179790916-14fef42e-cc57-4651-b0b0-f1e247cfb6dc.png)
   - After using explain analyse 
   - ![image](https://user-images.githubusercontent.com/70589265/179820857-4f82fd41-b482-4f15-a203-ce433f07d6d8.png)
- Ran this queries SELECT * FROM visits where vet_id = 2;
   - Before the explain analyse 
![analyze-2](https://user-images.githubusercontent.com/70589265/179791305-73a0af96-3546-4ed8-b308-4b4e048307ec.png)
   - After using the explain analyse 
![image](https://user-images.githubusercontent.com/70589265/179820996-9d9b5cc8-77ac-43a0-bf86-b282dc278efc.png)
- Ran this queries SELECT * FROM owners where email = 'owner_18327@mail.com';
   - Before using the explain analyse 
![analyze-3](https://user-images.githubusercontent.com/70589265/179791573-cd08de94-29b5-4f98-b9f9-a4266e298301.png)
   - After using the expalin analyse 
![image](https://user-images.githubusercontent.com/70589265/179821081-dc065bc1-aa1e-4f24-be73-9e1dae651bcc.png)